### DISCUSSION TITLE
can we(mruby) be an alternative to MicroPython?

### DISCUSSION THREAD

**Discussion by ghost** [mruby/mruby#6296] [Q&A] [answered]

**Comment by matz** [comment, marked as answer]
It depends on your project (and your preference). MicroPython has a bigger community thus making it easier for many projects.
But mruby has mrbgems that make extension easier. In addition, we have a lot of room to develop so that it's fun to work with.

If you are interested in mechanical keyboards, you may want to check out [picoruby](https://github.com/picoruby/picoruby/).
